setupext.py

```diff
@@ -1643,6 +1650,16 @@ class BackendTkAgg(OptionalBackendPackage):
     def check(self):
         return "installing; run-time loading from Python Tcl / Tk"
 
+    def runtime_check(self):
+        """ Checks whether TkAgg runtime dependencies are met
+        """
+        pkg_name = 'tkinter' if PY3min else 'Tkinter'
+        try:
+            __import__(pkg_name)
```

Comment:
I think you're right - but maybe worth noting that I'm not using `importlib`, only the `__import__` builtin.